syntax = "proto3";

package resdb.utxo;

import "proto/utxo/utxo.proto";

message GenesisUTXO {
  repeated UTXO transactions = 1;
}

message Config {
  GenesisUTXO genesis_transactions = 1;
}


